gc/base/TLHAllocationInterface.cpp

```diff
@@ -161,6 +161,22 @@ MM_TLHAllocationInterface::allocateFromTLH(MM_EnvironmentBase *env, MM_AllocateD
 	return result;
 };
 
+bool
+MM_TLHAllocationInterface::forceRefreshTLH(MM_EnvironmentBase *env, MM_AllocateDescription *allocDescription) {
+	bool result = false;
+
+#if defined(OMR_GC_NON_ZERO_TLH)
+	if (allocDescription->getNonZeroTLHFlag()) {
+		result = _tlhAllocationSupportNonZero.refresh(env, allocDescription, false, true);
+	} else
+#endif /* defined(OMR_GC_NON_ZERO_TLH) */ 
+	{
+		result = _tlhAllocationSupport.refresh(env, allocDescription, false, true);
+	}
+
+	return result;
+}
```

Comment:
If dual TLH mode is used this function is going to refresh one or another. So if intention is to refresh both TLHs either you should manage to do it here or alternatively call this function twice for each type of TLH